engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -199,11 +216,21 @@ class EngineJobExecutionActor(jobKey: BackendJobDescriptorKey,
     context.actorOf(props, s"ejha_for_$jobDescriptor")
   }
 
-  def lookupCachedResult(jobDescriptor: BackendJobDescriptor, cacheResultId: MetaInfoId) = {
-    // TODO: Start up a backend job copying actor (if possible, otherwise just runJob). That should send back the BackendJobExecutionResponse
-    self ! FailedNonRetryableResponse(jobKey, new Exception("Call cache result copying not implemented!"), None)
-    // While the cache result is looked up, we wait for the response just like we were waiting for a Job to complete:
-    goto(RunningJob) using EmptyPartialCompletionData
+  def lookupCacheResult(jobDescriptor: BackendJobDescriptor, bjeaProps: Props, taskOutputs: Seq[TaskOutput], cacheResultId: MetaInfoId) = {
+    context.actorOf(FetchCachedResultsActor.props(cacheResultId, taskOutputs))
+    goto(PreparingCachedOutputs) using JobDescriptorData(jobDescriptor, bjeaProps)
+  }
+
+  def copyCacheHit(jobDescriptor: BackendJobDescriptor, cachedJobOutputs: JobOutputs) = {
+    val props = factory.cacheHitCopyingActorProps(jobDescriptor, initializationData, serviceRegistryActor, cachedJobOutputs)
+    val cacheHitCopyingActor = context.actorOf(props, buildCacheHitCopyingActorName(jobDescriptor))
+    // Send an ExecuteJobCommand to kick off processing.  The referenced actor will actually be a job execution actor if
+    // this backend doesn't implement caching.
+    cacheHitCopyingActor ! ExecuteJobCommand
+    context.parent ! JobRunning(jobDescriptor, cacheHitCopyingActor) //send the same message as runJob to WorkflowExecutionActor
```

Comment:
If this is just for information (or just for metadata/events), I'd happily make a new `DecachingJobResult` alongside `JobRunning`, just to be more specific about what's going on